test/e2e/test_sandbox.py

```
@@ -147,6 +148,46 @@ def test_sandbox_dryrun_opencode_custom_image():
+def test_sandbox_dryrun_openclaw_custom_port():
+    """Custom --openclaw-port should be used by the OpenClaw gateway and TUI."""
+    result = check_output(_dryrun_cmd("openclaw") + ["--openclaw-port", "19001"])
+    assert "openclaw gateway run --port 19001 --bind loopback" in result
+    assert "openclaw tui --url ws://localhost:19001 --session main" in result
+
+
+@pytest.mark.e2e
+@skip_if_no_container
+def test_sandbox_dryrun_openclaw_state_dir():
+    """OpenClaw --state-dir should mount the host directory for gateway state."""
+    result = check_output(_dryrun_cmd("openclaw") + ["--state-dir", "/tmp/openclaw-state"])
```

Ruff (0.15.9)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)